style.css

```diff
@@ -997,8 +1003,634 @@ h2 {
   background: #6070ff;
 }
 
-@media screen and (min-device-width: 768px) {
-  header {
+@media screen and (min-width: 768px) {
```

Comment:
- [ ] Hello @joyapisi Great job so far :muscle: However I noticed that in an attempt to implement the previous reviewer's suggestion, you messed up with your desktop design and it all looks misarranged now. I have attached a video below for your reference. 
- [ ] I will suggest that you go back  and use the styles you had for your desktop design previously since they were perfect and also since that was the main requirement and resubmit for review so your project can be approved :+1: 

- Your Design now 
[Screencast from 02-24-2023 08:53:30 PM.webm](https://user-images.githubusercontent.com/102699958/221290029-dec9a1ba-5db6-405b-89b2-0ec97b791d1c.webm)

- Your Design before
[Screencast from 02-24-2023 08:55:26 PM.webm](https://user-images.githubusercontent.com/102699958/221290255-200e6ba2-d348-4251-a750-04352c4d8ed5.webm)